server/src/main/java/org/elasticsearch/inference/InferenceService.java

```diff
@@ -94,6 +95,7 @@ void infer(
         Model model,
         @Nullable String query,
         List<String> input,
+        boolean stream,
```

Comment:
Is the stream boolean passed in with every request to perform an inference? Is this something that can be part of task settings instead?

Reply:
Makes sense, I agree that this seems like the best option in the current system. Thanks for clarifying